index.html

```diff
@@ -0,0 +1,349 @@
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact-form">
+        <h2 class="contact-head">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+        <form action="" method="post">
+          <input type="text" name="full_name" placeholder="Full name">
+          <input type="email" name="email" placeholder="Email address">
+          <textarea name="message" cols="30" rows="10" placeholder="Write me something..."></textarea>
+          <input type="submit" value="Get in touch">
```

Comment:
Great job implementing the correct input types 👍 , however, there are some issues that need to be fixed.

- [ ] Kindly make sure you set the correct validation for them as per the requirements. You can do so for example by adding the `required` attribute. This would ensure every field is completed before users send their messages, and it would help your website to look more professional 💯 .

- [ ] Kindly make sure your full_name input and your textarea have their correct `maxlength` set as per the requirements (30 and 500 respectively). This will ensure setting a limit on the message size and is standard practice in the use of forms 👍 .

- [ ] Please make sure your submit button has all the effects from the Figam template like the rest of the buttons in your project (hover, pressed, etc). This is not only part of the requirements, but it will help your website to keep a consistent style.